src/templates/Avalonia.FuncUI.ElmishTemplate/content/.template.config/template.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Josua J�ger, Kevin Schneider",
+  "classifications": [
+    "Console",
+    "UI",
+    "Elmish"
+  ],
+  "name": "Elmish Template for Avalonia.FuncUI v0.1.0",
+  "tags": {
+    "language": "F#",
+    "type": "project"
+  },
+  "identity": "Avalonia.FuncUI.ElmishTemplate",
```

Comment:
Not sure about the name. 

I think 'Elmish' should be included in every template, so I would ditch the prefix and just call it 'basic'/'app' ?

In the future there could also be an 'Empty' template containing no view code, but has the rest already setup.

What do you think ? 🤔

Reply:
> I think 'Elmish' should be included in every template, so I would ditch the prefix and just call it 'basic'/'app' ?

Will do. Due to the name Avalonia.FuncUI i have a hard time coming up with something really short for the "shortName" field. Will not be an issue anymore when #11 is closed.

> In the future there could also be an 'Empty' template containing no view code, but has the rest already setup.

Speaking from experience using the safe stack template (which uses MVU with elmish to create webapps) it is very helpful to start with a small working example to get a hold of the MVU cycle. However, a --empty argument for the template should work.